specs/1118-ORDS-submit_order.md

```diff
@@ -12,3 +12,14 @@
 
 - I can see the raw offset price of the order (<a name="1118-ORDS-008" href="#1118-ORDS-008">1118-ORDS-008</a>)
 - I can see the reference price of the order (<a name="1118-ORDS-009" href="#1118-ORDS-009">1118-ORDS-009</a>)
+
+### Market Orders
+
+- I can see 'Market price' (<a name="1118-ORDS-013" href="#1118-ORDS-013">1118-ORDS-013</a>)
+
+### Enriched Orders
+
+- I see enriched data for the market (code) when data has been loaded appropriately (<a name="1118-ORDS-011" href="#1118-ORDS-011">)
+- I see enriched data for the order size when data has been loaded appropriately (<a name="1118-ORDS-012" href="#1118-ORDS-012">1118-ORDS-012</a>)
```

Comment:
I find the wording of all these ACs a little misleading. I think 'I see enriched data for the price' makes it sounds like we will see the price AND then further data that enriches it and gives more context, which is not true. Could we word those ACs like below?
'I see the order price in the enriched data view hen data has been loaded successfully'
'I see the order size in the enriched data view when data has been loaded successfully'